wsgi_kerberos.py

```diff
@@ -40,25 +40,39 @@ def _consume_request(environ):
     defending itself against denial-of-service attacks, or from badly broken
     client implementations.
     '''
+    if environ.get("HTTP_EXPECT") == "100-continue":
+        return
     try:
         sock = environ.get('wsgi.input')
         if hasattr(sock, 'closed') and sock.closed:
             return
-        # Figure out how much content is available for us to consume.
-        expected = int(environ.get('CONTENT_LENGTH', '0'))
+
+        content_length = environ.get('CONTENT_LENGTH')
+        if content_length is None:
```

Comment:
If you do leave the Exception printing, this should also cover an empty string. At least the wsgiref server converts a missing content-length header into an empty string.

Reply:
Thanks, handled in the latest revision.